common/src/main/java/com/wynntils/features/user/BeaconDistanceFeature.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.math.Matrix4f;
+import com.mojang.math.Vector3d;
+import com.mojang.math.Vector3f;
+import com.mojang.math.Vector4f;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.managers.Model;
+import com.wynntils.gui.render.FontRenderer;
+import com.wynntils.gui.render.HorizontalAlignment;
+import com.wynntils.gui.render.RenderUtils;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.event.RenderLevelEvent;
+import com.wynntils.mc.objects.CommonColors;
+import com.wynntils.mc.objects.CustomColor;
+import com.wynntils.mc.objects.Location;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wynn.model.CompassModel;
+import java.util.List;
+import net.minecraft.client.Camera;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class BeaconDistanceFeature extends UserFeature {
+
+    @Config
+    public CustomColor textColor = CommonColors.WHITE;
+
+    @Config
+    public int backgroundOpacity = 80;
+
+    @Config
+    public FontRenderer.TextShadow textShadow = FontRenderer.TextShadow.NONE;
+
+    private Vector3d ndc = null;
```

Comment:
Can you rename this to be more specific, so anyone reading can understand what this stores?

Reply:
@HalcyonSuoh I dont think you renamed the variable?

Reply:
I failed to find a more descriptive name myself, looking around codebase only to find other TextShadow is also called textShadow, `AuraTimerOverlayFeature` for instance.
as for ndc, which stands for Normalized Device Coordinates, which I also failed to find a appropriate name for, ditching the abbreviation also make the variable name way longer than it should be

Reply:
textShadow is fine, I didn't want you change that. But for ndc, I would rather have a long name, than a 3 char one.

Reply:
I doubt anyone seeing Normalized Device Coordinates know what it is or what it does, but I guess that make google experience smoother than the abbreviation